onmt/utils/loss.py

```diff
@@ -358,6 +382,9 @@ class LanguageModelLossCompute(CommonLossCompute):
     Standard LM Loss Computation.
     """
 
+    def _compute_alignement_loss(self, align_head, ref_align):
+        return 0
```

Comment:
Maybe explicitly say it's not enabled for LM. By the way didn't we have an assert at some point for lambda_align // LM case?

Reply:
Indeed lambda_align is asserted to be 0 + it will disappear if we agree on https://github.com/OpenNMT/OpenNMT-py/pull/1919#discussion_r529475321